manifests/metacontroller.yaml

```diff
@@ -70,3 +70,11 @@ spec:
         - --logtostderr
         - -v=4
         - --discovery-interval=20s
+        volumeMounts:
+        - mountPath: /etc/ssl/certs
+          name: ca-certs
+          readOnly: true
+      volumes:
+      - hostPath:
+          path: /etc/ssl/certs
+        name: ca-certs
```

Comment:
There should probably be a `type` here. The default empty string exists for backwards compatibility, but we should be more explicit about what checks we need.